src/Client/Search.php

```diff
@@ -68,17 +69,6 @@ public function forQuery(string $query)
         return $clone;
     }
 
-    public function forSubject(string ...$subjectId) : self
-    {
-        $clone = clone $this;
-
-        $clone->subjectsQuery = array_unique(array_merge($this->subjectsQuery, $subjectId));
-
-        $clone->invalidateDataIfDifferent('subjectsQuery', $this);
```

Comment:
it seems that https://github.com/elifesciences/api-sdk-php/pull/204/files#diff-85f6701f35e30712f7c04660ef7a7e06R204 changes 3 fields rather than just `count` like the `ForSubject` trait. Therefore I wouldn't use that trait here, remaining on the original implementation.